applications/e-commerce/Ankesh.html

```diff
@@ -0,0 +1,4 @@
+<h1>
+    Hello
+    Just checking pull request works or not ? :D
+</h1>
```

Comment:
Please remove this file from your PR. It's not part of the e-commerce application, and you already added a file in the `members` folder 😄

Reply:
Can you tell me how to remove it from PR ???